src/elements/e-picture.vue

```diff
@@ -272,6 +280,9 @@
       onLoad() {
         this.loaded = true;
       },
+      handleError() {
```

Comment:
todo: please prefix event handlers with `on` according to the HTML naming of e.g. `onclick=""`.

```suggestion
      onError() {
```